Library/Homebrew/formula.rb

```diff
@@ -2933,7 +2933,9 @@ def pour_bottle?(only_if: nil, &block)
         lambda do |_|
           on_macos do
             T.cast(self, PourBottleCheck).reason(+<<~EOS)
-              The bottle needs the Apple Command Line Tools to be installed.
+              The bottle needs the standalone Apple Command Line Tools to be installed. #{if MacOS::Xcode.installed?
+          "\nThe full version of Xcode is not compatible with this bottle."
```

Comment:
```suggestion
          "\nThe full version of Xcode is not sufficient to install this bottle."
```

I think something like this might be preferred (feel free to modify the exact wording). That way it's clear that having Xcode _is_ okay, it's just that you _also_ need the CLT.

Reply:
Actually, I was hoping for something like this:

With Xcode but no CLT (we can tweak the wording a bit):
```
You have an Xcode installation but the bottle needs the
standalone Command Line Tools to be installed. You can
install them, if desired, with:
  xcode-select --install
```

If you have neither Xcode nor CLT, then you see the existing message.

Reply:
Yeah, I like that better.

Do you think using the word "standalone" implies that Xcode needs to be uninstalled? I can't decide how I feel about it. I just don't want someone to read this and say "oh shoot I guess I need to replace Xcode with the CLT for this to work"

Reply:
I guess we can just say `Apple Command Line Tools` instead.

If we're extra worried we can add an FAQ about this message and include a link.

Reply:
How about this:

```
You have an Xcode installation. However, this bottle needs the standalone Command Line Tools, 
because the Xcode command line tools are not sufficient. You can
install them in addition to Xcode, if desired, with:
  xcode-select --install
```

Reply:
I'd rather make it less wordy. Most users don't read messages from `brew`, and making them wordier just means more users ignore them.

We can probably try to make it more like the existing message for a missing Xcode install (when one is required):

https://github.com/Homebrew/brew/blob/53710f8f4192905ce65060099f81fe1175f8fca0/Library/Homebrew/requirements/xcode_requirement.rb#L34-L37